packages/firebase/index.d.ts

```diff
@@ -4477,6 +4526,14 @@ declare namespace firebase.analytics {
     value?: number;
     event_label?: string;
     event_category: string;
+    shipping_tier?: string;
```

Comment:
The proposal only adds the following:
```
  // New suggested event parameters.
  shipping_tier?: string;
  item_list?: string;
  item_list_id?: string;
  payment_type?: string;
```
Where are the other ones defined?

Reply:
`item_list` is not present in the sheet. I think it has been replaced by `item_list_name`, see [comment](https://docs.google.com/document/d/1cJ0MsSbN6fzB36X9RHIZMuGJBQgqLxffXcgM0ofRIC4/edit?disco=AAAAJLaOVWQ).

Reply:
Removed.